docker/proxy/nginx.conf

```diff
@@ -23,6 +23,10 @@ server {
       proxy_pass http://alfresco:8080;
     }
 
+    location /api-explorer/ {
```

Comment:
why do we need api explorer for this compose file?

Reply:
You could use it as an easy way to see how to communicate directly with ACS REST API or even use it for direct communication with using the try functionality from api-explorer. As well api-explorer included in the alfresco/alfresco-content-repository-community image. It cam in handy for AGS as we need to figure out the API call for creating the Share RM side.